cmd/hkcam/main.go

```diff
@@ -63,6 +71,25 @@ func main() {
 		MultiStream:      *multiStream,
 	}
 
+	// Add Motion Sensor
+	if motionPin != -1 {
+		err := rpio.Open()
+		if err != nil {
+			log.Info.Panic(err)
+		}
+		pin := rpio.Pin(10)
```

Comment:
I think that must be:
```suggestion
		pin := rpio.Pin(motionPin)
```